Src/AutoFoqUnitTest/FoqMethodQueryTest.fs

```diff
@@ -74,7 +89,10 @@ let SelectMethodsDefineCorrectParameters (request: Type) =
             ||| BindingFlags.Instance
             ||| BindingFlags.NonPublic)         
         |> Seq.map(fun ci -> ci.GetParameters())
-    let sut = FoqMethodQuery()
+    let dummyBuilder =

```

Comment:
Done in [ef5280e](https://github.com/moodmosaic/AutoFixture/commit/ef5280ed4a30827caaf1ee33999937b7b9c7781a).